charts/posthog/templates/kafka-service.yaml

```diff
@@ -1,21 +0,0 @@
-{{- if .Values.kafka.service.enabled }}
```

Comment:
Not sure why this was added in the first place but we have a built-in service available with the upstream chart.